### ISSUE TITLE
Hosting with 8 players+ results in crash on Linux

### ISSUE THREAD

**Issue by insunaa** [MNoya/PMP#5]
Hey there, love the mod you did a great job.
I don't know if this is caused by the mod or by the linux version of DotA, but whenever I host a lobby of 8 or more players, it works for a few seconds, but when one of my peons attacks an enemy barricade (which may be entirely unrelated), my game immediately segfaults. Unfortunately the console just says segfault and no further info and I don't want to upload the crashdump on github, as I don't know if it contains private information.

**Comment by MNoya** [comment]
I wanted to remove the barricades I believe they are the cause of some of those weird crashes. Does it happen consistently? Does it happen with less than 8 players? Any additional info can help me debug it, although I should probably just remove barricades anyway (they don't serve much gameplay purpose after all)

**Comment by insunaa** [comment]
There have not been many players playing recently. It's happened to me around 5 times, all of the times were either 8 or 9 people, but I've not hosted a lobby with 7. I have hosted a lobby of 6 without problems, tho.
It has happened in all games I hosted with 8 or 9 people, but not with less than 7.

**Comment by MNoya** [comment]
Try it now with dedicated servers

**Comment by insunaa** [comment]
OK I just tried and it still happens. This time it was a 6 player lobby and my game crashed when I attacked a barrier. The server stayed operational, tho.
In addition to that there's now a weird bug, where my peons chase after dead enemy night elves, when their corpses are flung far away, so they try to attack a corpse and thus end up dying because all living enemies are still attacking my now passive peons.
